src/webxdc.rs

```diff
@@ -111,9 +112,9 @@ pub struct WebxdcInfo {
     /// Address to be used for `window.webxdc.selfAddr` in JS land.
     pub self_addr: String,
 
-    /// Milliseconds to wait before calling `sendUpdate()` again since the last call.
+    /// Time to wait before calling `sendUpdate()` again since the last call.
     /// Should be exposed to `window.sendUpdateInterval` in JS land.
-    pub send_update_interval: usize,
+    pub send_update_interval: Duration,
```

Comment:
I hope i don't break things here? This is `Serialize`, but it seems that the true public API is the JSON-RPC one

Reply:
There's already `test_webxdc()`, but it tests `WebxdcMessageInfo` defined in `deltachat-jsonrpc/src/api/types/webxdc.rs`, not this struct.

Still, this `WebxdcInfo` is public, so maybe there are some other implementations or core bindings using it, that's why i asked.